packages/p2p/src/components/order-details/order-details.scss

```diff
@@ -54,6 +54,18 @@ $card-width: 456px;
         width: 100%;
     }
 
+    &--rating {
+        margin: 2.4rem;
+
+        @include mobile {
+            margin: 1.6rem;
+        }
+
+        &__text {
+            margin: 0em 2.4rem;
```

Comment:
`nit` should this be like `margin: 0 2.4rem`, or alternatively `margin: unset 2.4rem`?